howso/engine/session.py

```diff
@@ -36,13 +32,23 @@ class Session(BaseSession):
         The Howso client instance to use.
     """
 
+    attribute_map = {
+        'id': 'id',
+        'name': 'name',
+        'user': 'user',
+        'metadata': 'metadata',
+        'created_date': 'created_date',
+        'modified_date': 'modified_date'
+    }
+
     def __init__(
         self,
         name: Optional[str] = None,
         *,
         id: Optional[str] = None,
         metadata: Optional[dict] = None,
         client: Optional[AbstractHowsoClient] = None,
+        **kwargs
```

Comment:
Shouldn't need to add kwargs here

Reply:
Should be addressed by: d66654ea10038f70a275bb4aa504128552745e60